packages/design-mui/src/components/MuiList/components/MuiBody.tsx

```diff
@@ -1,6 +1,6 @@
 import { Checkbox, TableBody, TableCell, TableRow } from "@mui/material"
 import type { HatchifyDisplay, XCollectionProps } from "@hatchifyjs/react-ui"
-import MuiBodySkeleton from "./MuiBodySkeleton"
+import { MuiBodySkeleton } from "./MuiBodySkeleton"
```

Comment:
this should be a default import i think

Reply:
yeah, you're right. There was a warning that I misunderstood